src/api/Gemfile.lock

```diff
@@ -226,6 +228,10 @@ GEM
     responders (2.2.0)
       railties (>= 4.2.0, < 5.1)
     riddle (1.5.12)
+    rspec (3.5.0)
+      rspec-core (~> 3.5.0)
+      rspec-expectations (~> 3.5.0)
+      rspec-mocks (~> 3.5.0)
```

Comment:
I just recreate my vagrant machine from the scratch, rebase to master, and running `vagrant exec bundle install`... it appeared